src/application.ts

```diff
@@ -9,6 +9,7 @@ import {
 } from "./types";
 import pick from "./utils/pick";
 import unpick from "./utils/unpick";
+import { finished } from "stream";
```

Comment:
@spersico This seems to be an unexpected import? I don't see it being used anywhere.